gce_workload_cert_refresh/main.go

```diff
@@ -279,11 +286,24 @@ func refreshCreds() error {
 		return fmt.Errorf("Error creating temporary link: %v", err)
 	}
 
+	oldTarget, err := os.Readlink(symlink)
+	if err != nil {
+		logger.Infof("Error reading existing symlink: %v\n", err)
+		oldTarget = ""
```

Comment:
No need to initialize `oldTarget` here, `Readlink()` will return an empty string in case of error.

Reply:
ah i am probably leaning on old guidelines here, but the principle is that functions which return error shouldn't be trusted with any of their other return values